### PR TITLE
Update MechanicAcceptance page in manager.

### PR DESCRIPTION
**Pull request by diyorzz**
Before:
Screenshot 2024-10-13 150112
After:
image

### CODE REVIEW

**Comment by Boburm1rzo** [comment]
Do we need a "Mexanik (Qabul qiluvchi)" header?
I think removing this is a good option

**Comment by diyorzz** [comment]
> Do we need a "Mexanik (Qabul qiluvchi)" header? I think removing this is a good option

"Yes, this is what we need, because we have written all the pages, and the information on the sidebar may not be enough, as older adults work there."

**Review by Boburm1rzo — APPROVED** [review]
nice

**Comment by jamshidbek7375** [comment]
Do we need to display data from all columns in the list? For example, "Acceptance distance" and "Remaining fuel".

**Comment by jamshidbek7375** [comment]
I think it would be better if you set the data in the "qabul qilish" column to be the same as the "status" column. That is, giving colors.

**Comment by diyorzz** [comment]
> Do we need to display data from all columns in the list? For example, "Acceptance distance" and "Remaining fuel".

No, we don't need to display data from all columns in the list. We can choose specific columns that are relevant or necessary for the current context.

**Comment by diyorzz** [comment]
> I think it would be better if you set the data in the "qabul qilish" column to be the same as the "status" column. That is, giving colors.

Now, I don't need to add color to "Qabul qilish" column because we won't display it on the main page.

**Comment by jamshidbek7375** [comment]
> > I think it would be better if you set the data in the "qabul qilish" column to be the same as the "status" column. That is, giving colors.
> 
> Now, I don't need to add color to "Qabul qilish" column because we won't display it on the index. we display on details page.

Good great

File: CheckDrive.Web/CheckDrive.Web/Views/Accounts/Index.cshtml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     ViewData["Title"] = "Index";
     Layout = "~/Views/Shared/_Layout.cshtml";
 }
-<h2>Ishchilar</h2>
+<h1>Ishchilar</h1>
```
Comment (jamshidbek7375) [inline CheckDrive.Web/CheckDrive.Web/Views/Accounts/Index.cshtml:7]: The condition is to do h1. h2 is fine.

**Review by jamshidbek7375 — APPROVED** [review]

File: CheckDrive.Web/CheckDrive.Web/Views/Accounts/Index.cshtml (hunk 1 above)
Comment (diyorzz) [reply CheckDrive.Web/CheckDrive.Web/Views/Accounts/Index.cshtml:7]: We do not use this line of code now.

**Review by FirdavsAX — COMMENTED** [review]
Is this pull request need code review , is this branch merged

**Review by Ramadan1011 — APPROVED** [review]
looks good

**Review by SharifovDeveloper — APPROVED** [review]

**Pull request merged**